Classes/Emogrifier.php

```diff
@@ -102,6 +102,26 @@ class Emogrifier
     private $styleAttributesForNodes = array();
 
     /**
+     * Determines whether the "style" attributes of tags in the the HTML passed to this class should be preserved.
+     * If set to false, the value of the style attributes will be discarded.
+     *
+     * @var bool
+     */
+    private $isInlineStylesParsingEnabled = true;
```

Comment:
@oliverklee I like your suggestion. Are you thinking:
`isInlineStyleAttributesParsingEnabled`
or 
`isStyleAttributesParsingEnabled`
I prefer the former, but just wanted to check.